src/Microsoft.OpenApi.Readers/V2/OpenApiSecurityRequirementDeserializer.cs

```diff
@@ -33,8 +33,8 @@ public static OpenApiSecurityRequirement LoadSecurityRequirement(ParseNode node)
                 }
                 else
                 {
-                    node.Diagnostic.Errors.Add(
-                        new OpenApiError(node.Context.GetLocation(), $"Scheme {property.Name} is not found"));
+                    //node.Diagnostic.Errors.Add(
```

Comment:
What should happen if `scheme` is `null`?

Reply:
The spec says 

> the name used for each property MUST correspond to a security scheme declared in the Security Schemes under the Components Object.
https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.1.md#security-requirement-object

So I believe we should mark this as an error.  Not sure why this has been commented out /cc @PerthCharern

Reply:
I commented this out so that I could test without running into NRE and forgot to revert back. 

---
In reply to: [165090553](https://github.com/Microsoft/OpenAPI.NET/pull/197#discussion_r165090553) [](ancestors = 165090553)